### ISSUE TITLE
[DEV-1174] Nav bar contrast and expand/collapse

### ISSUE THREAD

**Issue by angelinanz** [hathitrust/firebird-common#73]
[DEV-1174]

[1727975](https://axeauditor.dequecloud.com/test-run/b2613a94-fe0b-11ee-96ce-dfcb37e53e3c/issue/0cb64ae0-ff1d-11ee-934f-6ff1ae8d2d9c?sortField=severity&sortDir=desc&page=0&issuesCount=&filter%5Bcomponent_number%5D=1&row=3) focus indicator lacks contrast
 [1727026](https://axeauditor.dequecloud.com/test-run/b2613a94-fe0b-11ee-96ce-dfcb37e53e3c/issue/302de104-fe0f-11ee-ac5b-ff3565d28288?sortField=severity&sortDir=desc&page=0&issuesCount=&filter%5Bcomponent_number%5D=1&row=5) expand/collapse on search bar isn't setting aria expanded correctly

[DEV-1174]: https://hathitrust.atlassian.net/browse/DEV-1174?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
